main.js

```diff
@@ -0,0 +1,39 @@
+const submitButton = document.getElementsByClassName('active')[0];
+
+submitButton.addEventListener('click', function () {
+
+const inputPost = document.getElementsByClassName('post')[0].value;
+const inputName = document.getElementsByClassName('name')[0].value;
+
+const newSpan = document.createElement('span');
+newSpan.innerHTML = `${inputPost}, posted by ${inputName}`;
+const lineBreak = document.createElement('br');
+
+const commentContainer = document.querySelector('.new-comments');
+commentContainer.appendChild(newSpan);
+commentContainer.appendChild(lineBreak);
+});
+
+const rmButton = document.getElementsByClassName('rm-comments-btn')[0];
+rmButton.addEventListener('click', function () {
+  const commentsContainer = document.querySelector('.new-comments');
+  commentsContainer.innerHTML = '';
+})
+
+const toggleButtons = document.querySelectorAll('.show-comments-btn');
+
+
+toggleButtons.forEach(button => {
```

Comment:
this should have been a separate function with ternary operation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator
```
function toggleVisibility(element) {
  element.style.display = element.style.display === 'none' ? 'block' : 'none';
}
```